Makefile

```diff
@@ -0,0 +1,10 @@
+all: clean build-collector test-collector
+
+build-collector:
```

Comment:
Sure! I wasn't sure if it would ever be desirable to build these individually, but yeah I think `build` and `test` will be simpler for now. I'll push up a commit that changes this.